front/src/applications/stdcm/components/StdcmResults/SimulationReportSheet.tsx

```diff
@@ -116,6 +116,10 @@ const SimulationReportSheet = ({
                 <Text style={styles.convoyAndRoute.convoyInfoData}>
                   {`${Math.floor(convoyMaxSpeed)} km/h`}
                 </Text>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('loadingGauge')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>
+                  {consist?.loadingGauge ?? rollingStock.loading_gauge}
```

Comment:
We always have a `loadingGauge` selected now with the new select
```suggestion
                  {consist?.loadingGauge}
```

Reply:
Done